kalite/coachreports/features/environment.py

```diff
@@ -1,4 +1,4 @@
 """
 environment.py specific to the this app
 """
-from kalite.testing.base_environment import before_all, after_all, before_feature, after_feature
+from kalite.testing.base_environment import *
```

Comment:
laziness can be a valid argument :)

Laziness is a good reason for PyCharm or LiClipse btw :)